hypothesis-python/RELEASE.rst

```diff
@@ -0,0 +1,3 @@
+RELEASE_TYPE: minor
```

Comment:
Last tweak sorry - this should be `RELEASE_TYPE: patch`, because it's semver-compatible with the current minor version.

Otherwise this looks great!  We'd love to have you on the list of authors in `CONTRIBUTING.rst`, FYI 😄